components/guidance_feedback/__snapshots__/guidance-feedback.react.test.js.snap

```diff
@@ -0,0 +1,311 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Hide the form and button elements on submit 1`] = `
+<form
+  action=""
+  className="react_component_gf"
+  id="guidance_feedback"
+>
+  <h2>
+    Feedback
+  </h2>
+  <h3
+    id="aria"
+  >
+    Did you find the guidance you needed?
+  </h3>
+  <button
+    aria-describedby="aria"
```

Comment:
This should have an `aria-label="No"`
Please refer to link [here](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/ARIA_Techniques/Using_the_aria-label_attribute)

Reply:
Resolved